README.md

```diff
@@ -54,18 +55,14 @@ For the given number of CIDs and the number of workers, the tool will automatica
 
 For example, if we want to generate 1000 CIDs with a worker number of 200 CIDs, the tool will spawn 200 workers to publish the CIDs concurrently.
 
-_NOTE: The Provider Records WON’T be republished after 12 hours. The goal of the study is to track the theoretical record lifetime for a range of CIDs across the hash space._
-
-## Discoverer
+_NOTE: The Provider Records WONΓÇÖT be republished after 12 hours. The goal of the study is to track the theoretical record lifetime for a range of CIDs across the hash space._ TODO Maybe change this to 22H as of late versions
 
-The discoverer reads a set of cids from a json file, instead of generating them itself. Right now the discoverer can only track already published cids in the network and won't be able to publish them
-itself. Then it proceeds to follow the exact same steps as the publisher.
 
 ## Pinger
 
 Once each CID has been generated and published to the network or read from a file, The tool proceeds to persist to the DB the recorded information and sends the CID information to the CID Ping-Orchester routine.
 
-The CID Ping-Orchester has a list of CIDs ordered by the next connection time. It uses this next connection time to determine whether the CID needs to be pinged again or not. If the CID’s next connections proves that the track frequency has passed since the last ping, it will add the CID info into a `Ping Queue` for the ping worker.
+The CID Ping-Orchester has a list of CIDs ordered by the next connection time. It uses this next connection time to determine whether the CID needs to be pinged again or not. If the CIDΓÇÖs next connections proves that the track frequency has passed since the last ping, it will add the CID info into a `Ping Queue` for the ping worker.
```

Comment:
Can we leave the `'` character as it was?
```suggestion
The CID Ping-Orchester has a list of CIDs ordered by the next connection time. It uses this next connection time to determine whether the CID needs to be pinged again or not. If the CID’s next connection proves that the tracking frequency has passed since the last ping, it will add the CID info into a `Ping Queue` for the ping worker.
```